### PR TITLE
BLD: update conda recipe

### PR DESCRIPTION
**Pull request by tangkong**
## Description
Update recipe, at the least the run requirements should be in sync with the conda feedstock

## Motivation and Context
I get confused every time I see a conda feedstock pr

## How Has This Been Tested?
ci I choose you

## Where Has This Been Documented?
This PR

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Code has been checked for threading issues (no blocking tasks in GUI thread)
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran ``docs/pre-release-notes.sh`` and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Comment by tangkong** [comment]
Only really to be merged once we figure out what we want for https://github.com/conda-forge/typhos-feedstock/pull/29, this and that should agree

**Comment by ZLLentz** [comment]
My opinion is that this is essentially correct
Do we use pcdsdevices at all in the test suite? Maybe we need that in the test dependencies

File: .github/workflows/standard.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
       # Extras that will be installed for both conda/pip:
       testing-extras: "happi"
       # Extras to be installed only for conda-based testing:
-      conda-testing-extras: "pcdsdevices pyqt=5"
+      conda-testing-extras: "pyqt=5"
```
Comment (klauer) [inline .github/workflows/standard.yml:21]: This can be removed since it's in conda recipe requirements

File: .github/workflows/standard.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ jobs:
...
       # Extras to be installed only for pip-based testing:
       pip-testing-extras: "PyQt5<5.15"
```
Comment (klauer) [inline .github/workflows/standard.yml:23]: We should make sure this is in `requirements.txt` for pip-based installs (`PyQt5` no pin)

File: .github/workflows/standard.yml (hunk 1 above)
Comment (tangkong) [reply .github/workflows/standard.yml:21]: 🤦 oh yes

File: .github/workflows/standard.yml (hunk 2 above)
Comment (tangkong) [reply .github/workflows/standard.yml:23]: `PyQt5` is in `requirements.txt` 👍

Comment (klauer) [reply .github/workflows/standard.yml:23]:
```suggestion
      pip-testing-extras: ""
```

**Review by ZLLentz — APPROVED** [review]
I'm fairly certain this is correct

**Comment by tangkong** [comment]
I'm going to merge this before we (I) forget about it again. Hopefully I'm less confused next time tagging comes around

**Pull request merged**